CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## [Unreleased]
 
+### Added
+
+-
+
+### Fixed
+
+- `get_mlflow_config` now uses the kedro context config_loader to get configs (#66 #64 #54 #30 #31 #72)
+
+### Changed
+
+- `MlflowNodeHook` have now a before_pipeline_run hook allowing to load the kedro context, get configs and populating MlflowNodeHook attributes
+- `mlflow init` is now getting conf path from `context.CONF_ROOT`
+- Change conf_dir fixture in conftest by adding `.kedro.yml`, a `src/dummy_package_run.py` tree structure and `globals.yml`.
+- Change tests that are impacted by the introduction of the kedro context
```

Comment:
I don't think that tests modifications go the CHANGELOG which is intended for users (but it's nice for the review to tell what you changed ;) I think it rather belongs to the discussion in the PR.

Reply:
Ok for me